logger/logger.go

```diff
@@ -0,0 +1,47 @@
+package logger
+
+import (
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-go-client/internal/hzerrors"
+)
+
+// Logger is the interface that is used by client for logging.
+// Level is: type Level string
```

Comment:
This documentation seems like out-of-date. 
We should describe what Weight is instead here I guess

Reply:
Thanks, great catch.